Tests/Tests.cs

```diff
@@ -208,79 +261,13 @@ public void TriggerChallenges()
                     .WithPath("/validate/triggerchallenge")
                     .UsingPost()
                     .WithBody("user=test")
-                    .WithHeader("Authorization", authToken)
+                    .WithHeader("Authorization", this.authToken)
```

Comment:
will be _AuthToken. Can drop the `this`

Reply:
[98fb81a](https://github.com/privacyidea/csharp-client/pull/10/commits/98fb81a65433bb4a424fab9883efcc50ad39cfd4)